### ISSUE TITLE
high vulnerability in dependent library axios

### ISSUE THREAD

**Issue by jolly-jump** [ether/ueberDB#168]
Hi, 
sorry, I don't know much about the structure of nodejs, so I wonder how this audit could be fixed:

```
npm list axios
`-- ep_etherpad-lite@1.8.7 -> /opt/etherpad-lite/src
  `-- ueberdb2@0.5.6
    `-- mssql@6.3.0
      `-- tedious@6.7.0
        `-- @azure/ms-rest-nodeauth@2.0.2
          `-- @azure/ms-rest-js@1.9.0
            `-- axios@0.19.2 
```

where axios has a security problem when using proxies: https://www.npmjs.com/advisories/1594

**Comment by jolly-jump** [comment]
I guess that the master-branch of mssql@7.0.0-alpha4 would pull in a supported and thus fixed version of azure-ms-rest-nodeauth, etc...

**Comment by JohnMcLear** [comment]
ja gimme 5 mins , im fixing.

**Comment by JohnMcLear** [comment]
Fixed in 1.1.7

**Comment by jolly-jump** [comment]
Yes, this works for me. Thank you, @JohnMcLear .
